virtcontainers/device/manager/utils.go

```diff
@@ -35,3 +40,92 @@ func isVFIO(hostPath string) bool {
 func isBlock(devInfo config.DeviceInfo) bool {
 	return devInfo.DevType == "b"
 }
+
+func IsVFIOLargeBarSpaceDevice(hostPath string) (bool, error) {
+	if !isVFIO(hostPath) {
+		return false, nil
+	}
```

Comment:
Nit: Can you add some blank lines between blocks to allow the code to "breathe"? It looks rather cramped like this.

Reply:
ok